mne_bids/mne_bids.py

```diff
@@ -29,13 +29,16 @@
 
 
 ALLOWED_KINDS = ['meg', 'ieeg']
-orientation = {'.sqd': 'ALS', '.con': 'ALS', '.fif': 'RAS', '.pdf': 'ALS',
+ORIENTATION = {'.sqd': 'ALS', '.con': 'ALS', '.fif': 'RAS', '.pdf': 'ALS',
                '.ds': 'ALS'}
 
-units = {'.sqd': 'm', '.con': 'm', '.fif': 'm', '.pdf': 'm', '.ds': 'cm'}
+UNITS = {'.sqd': 'm', '.con': 'm', '.fif': 'm', '.pdf': 'm', '.ds': 'cm'}
 
-manufacturers = {'.sqd': 'KIT/Yokogawa', '.con': 'KIT/Yokogawa',
-                 '.fif': 'Elekta', '.pdf': '4D Magnes', '.ds': 'CTF'}
+MANUFACTURERS = {'.sqd': 'KIT/Yokogawa', '.con': 'KIT/Yokogawa',
+                 '.fif': 'Elekta', '.pdf': '4D Magnes', '.ds': 'CTF',
+                 '.meg4': 'CTF'}
+
+IGNORED_CHANNELS = {'KIT/Yokogawa': ['STI 014']}
```

Comment:
please add a comment that this is only to be used to ignore stimulus channels (for now).